dashboard/app/models/pd/enrollment.rb

```diff
@@ -172,7 +172,7 @@ def self.filter_for_survey_completion(enrollments, select_completed = true)
     foorm_enrollments, other_enrollments = enrollments.partition do |enrollment|
       (enrollment.workshop.workshop_ending_date >= Date.new(2020, 5, 8) &&
         (enrollment.workshop.csf_intro? || enrollment.workshop.local_summer? || enrollment.workshop.csp_wfrt?)) ||
-        (enrollment.workshop.workshop_ending_date >= Date.new(2020, 9, 1) && enrollment.workshop.csf_201?)
+        (enrollment.workshop.workshop_ending_date >= Date.new(2020, 9, 1) && enrollment.workshop.csf_201?) || enrollment.workshop.csf_district?
```

Comment:
No need for a workshop_ending_date here?  When was it switched to `foorm`? Also, could you update the comment above this method?